doc/source/docs/howto_build_docs.rst

```diff
@@ -4,76 +4,100 @@
 Building the NumPy API and reference docs
 =========================================
 
-We currently use Sphinx_ for generating the API and reference
-documentation for NumPy.  You will need Sphinx >= 2.2.0.
-
 If you only want to get the documentation, note that pre-built
 versions can be found at
 
     https://numpy.org/doc/
 
 in several different formats.
 
-.. _Sphinx: http://www.sphinx-doc.org/
 
+Prerequisites
+-------------
 
-Instructions
-------------
+Before proceeding further it should be noted that the documentation is built with the ``make`` tool,
+which is not available on Windows. It is recommended for Windows users to set up their development
```

Comment:
Maybe we can qualify this as "not natively available on Windows" or similar. There are a number of solutions to using make on windows and they might work, we just don't want to recommend any in particular.